tests/attestation_cert_apis/host/host.cpp

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#define FILENAME_LEN 80
```

Comment:
Per dev guidelines, prefer verbose names over abbreviations and acronyms: `FILENAME_LENGTH`